docs/personal-designs.rst

```diff
@@ -0,0 +1,148 @@
+Compiling and Running Your Own Designs Using the Symbiflow-examples Toolchain 
+==============================================================================
+
+This section describes how to compile and download your own designs to an FPGA using only the symbiflow-examples tool chain.
+
+Setup Prior to Running the Toolchain
+--------------------------------------
+Before building any examples, you will need to first install the toolchain. To do this, follow the steps in `Getting Symbiflow <getting-symbiflow.html>`_.
+After you have downloaded the toolchain, set the installation directory to match what you set it to earlier, for example:
+
+.. code-block:: bash
+   :name: export-install-dir
+
+   export INSTALL_DIR=~/opt/symbiflow
+
+Select your FPGA family:
+
+.. tabs::
+
+   .. group-tab:: Artix-7
+
+      .. code-block:: bash
+         :name: fpga-fam-xc7
+
+         FPGA_FAM="xc7"
+
+   .. group-tab:: EOS S3
+
+      .. code-block:: bash
+         :name: fpga-fam-eos-s3
+
+         FPGA_FAM="eos-s3"
+
+Next, set the path and source for your conda environment:
+
+.. code-block:: bash
+   :name: conda-prep-env
+
+   export PATH="$INSTALL_DIR/$FPGA_FAM/install/bin:$PATH";
+   source "$INSTALL_DIR/$FPGA_FAM/conda/etc/profile.d/conda.sh"
+
+Finally, activate your Conda environment:
+
+.. code-block:: bash
+   :name: conda-act-env
+
+   conda activate $FPGA_FAM
+
+
+.. note::
+
+   You will need to run the commands for setting the path and source of your conda environment each time you open a new terminal. You will also need to activate the Conda environment for your hardware before you attempt to build your designs. It might be a good idea to add the above commands to your ``.bashrc`` either as default commands that run each time you open a new terminal or aliases to save yourself some repetitive typing. 
+
+
+Preparing Your Design 
+----------------------
+Building a design in symbiflow requires three parts, the HDL files for your design, a constraints file, and a Makefile. For simplicity, all three of these design files should be moved to a single directory. The location of the directory does not mater as long as the three design files are all within it.
+
+HDL Files
+++++++++++
+Symbiflow provides support for both Verilog and SystemVerilog HDL code. Use whichever method you prefer and add your design files to the directory of choice. If you are using the provided Makefiles to build your design, the top level module in your HDL code should be declared as ``module top (...``. Failure to do so will result in an error during the build process stating something similar to ``ERROR: Module 'top' not found!``
```

Comment:
As above, SystemVerilog is not fully supported.

The statement about the `top` module is not entirly correct, as for instance, `symbiflow_synth` has an argument to define [the top-level module name](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/39de38b90ed61fd3fd6e162f8296beeef322759f/xc/xc7/toolchain_wrappers/symbiflow_synth#L31-L38)